packages/arb-token-bridge-ui/src/hooks/__tests__/useBalance.test.tsx

```diff
@@ -3,12 +3,12 @@
  */
 
 import { act, renderHook } from '@testing-library/react-hooks'
-import { useBalance } from '../useBalance'
 import { StaticJsonRpcProvider } from '@ethersproject/providers'
 import { BigNumber } from 'ethers'
 import { SWRConfig } from 'swr'
 import { PropsWithChildren } from 'react'
 import { MultiCaller } from '@arbitrum/sdk'
+import { useBalance } from './../useBalance'
```

Comment:
Why this change?

Reply:
Sorted the imports according to absolute + relative.